docs/glossary.md

```diff
@@ -33,6 +33,7 @@ The following is a glossary of domain specific terminology. Although benchmarks
 
 ## Analysis
 
+* **artifact comparisons**: the comparison of two artifacts. This is composed of many test result comparisons. The [comparison page](https://perf.rust-lang.org/compare.html) shows a single artifact comparison between two artifacts.
 * **test result comparison**: the delta between two test results for the same test case but (optionally) different artifacts. The [comparison page](https://perf.rust-lang.org/compare.html) lists all the test result comparisons as percentages between two runs.  
```

Comment:
The "(optionally) different artifacts" here confuses me. It would be pretty unusual to compare one version of rustc against itself, no?

Reply:
This is true. We don't enforce that they're different, but I guess we can just remove this.